PolyEngine/RenderingDevice/OpenGL/Src/Pipeline/EnvCapture.cpp

```diff
@@ -258,12 +256,12 @@ void EnvCapture::CaptureSpecularPrefilteredMap()
 
 	Matrix ViewFromModel[] =
 	{
-		Matrix(Vector::ZERO, Vector(1.0f,  0.0f,  0.0f), Vector(0.0f, -1.0f,  0.0f)),
+		Matrix(Vector::ZERO, Vector( 1.0f,  0.0f,  0.0f), Vector(0.0f, -1.0f,  0.0f)),
```

Comment:
Why is this `ViewFromModel` repeated three times? It should be moved to some helper function or to static global constant.

Reply:
Moved to common variable.